src/python/gudhi/weighted_rips_complex.py

```diff
@@ -0,0 +1,60 @@
+# This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+# See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+# Author(s):       Raphaël Tinarrage, Yuichi Ike, Masatoshi Takenouchi
+#
+# Copyright (C) 2020 Inria, Copyright (C) 2020 FUjitsu Laboratories Ltd.
+#
+# Modification(s):
+#   - YYYY/MM Author: Description of the modification
+
+from gudhi import SimplexTree
+
+class WeightedRipsComplex:
+    """
+    class to generate a weighted Rips complex 
+    from a distance matrix and weights on vertices
+    """
+    def __init__(self, 
+                distance_matrix, 
+                weights=None,
+                max_filtration=float('inf')):
+        """
+        Parameters:
+            distance_matrix: list of list of float,
```

Comment:
The syntax of your comments does not match what sphinx+napoleon expect (https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html). The closest would be to remove `:` after "Parameters", underline that word, and remove the comma after the type.